src/NexusMods.App.UI/Controls/Diagnostics/DiagnosticEntryView.axaml

```diff
@@ -6,23 +6,30 @@
                                 xmlns:diagnostics="clr-namespace:NexusMods.App.UI.Controls.Diagnostics"
                                 xmlns:icons="clr-namespace:NexusMods.Icons;assembly=NexusMods.Icons"
                                 xmlns:navigation="clr-namespace:NexusMods.App.UI.Controls.Navigation"
-                                mc:Ignorable="d" d:DesignWidth="980" d:DesignHeight="60"
+                                xmlns:panels="clr-namespace:Avalonia.Labs.Panels;assembly=Avalonia.Labs.Panels"
+                                mc:Ignorable="d" d:DesignWidth="500" d:DesignHeight="100"
                                 x:Class="NexusMods.App.UI.Controls.Diagnostics.DiagnosticEntryView">
+    
     <Design.DataContext>
         <diagnostics:DiagnosticEntryDesignViewModel />
     </Design.DataContext>
 
-    <navigation:NavigationControl x:Name="EntryButton" HorizontalAlignment="Stretch" Padding="12">
-        <StackPanel Orientation="Horizontal"
-                    HorizontalAlignment="Stretch"
-                    Spacing="{StaticResource Spacing-2}"
-                    VerticalAlignment="Center">
+    <Border x:Name="DropShadowBorder" VerticalAlignment="Center" Height="52">
+        <navigation:NavigationControl x:Name="EntryButton" VerticalAlignment="Stretch" HorizontalAlignment="Stretch" Padding="12,0,4,0">
+            
+                <panels:FlexPanel x:Name="EntryFlexPanel" Direction="Row" ColumnSpacing="8">
+                    <icons:UnifiedIcon x:Name="SeverityIcon" Size="20" HorizontalAlignment="Left" />
+                    <TextBlock x:Name="DescriptionText"
+                               Theme="{StaticResource BodyMDNormalTheme}"
+                               VerticalAlignment="Center"
+                               TextTrimming="CharacterEllipsis"
+                               TextWrapping="NoWrap"
+                               />
+                    <icons:UnifiedIcon Classes="ChevronRight" Size="32" HorizontalAlignment="Right" VerticalAlignment="Center"/>
+                </panels:FlexPanel>
```

Comment:
You shouldn't be using Horizontal and vertical alignment on the child elements when using `FlexPanel` as it basically overrides the FlexPanel own layout.

To get the distribution you want you should try playing with `JustifyContent` property on the FlexPanel. It determines the distribution of items along the main axis (horizontal in this case).

I think SpaceBetween is likely what we want with Flex.Grow=1 on the TextBlock to allow it to take up as much space in the middle as available.